sites/public/cypress/integration/pages/submit-application.spec.ts

```diff
@@ -3,8 +3,9 @@ import { coliseumApplication, minimalDataApplication } from "../../mockData/appl
 describe("Submit", function () {
   it("should submit an application for the Test: Coliseum listing", function () {
     cy.submitApplication("Test: Coliseum", coliseumApplication, false)
+    cy.signOut()
```

Comment:
Curious why we changed this testing flow here

Reply:
this is the same case as above with the cypress tests, we don't have to have code that logs us out though, pulled that out